tsdb/importer/util.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2019 The Prometheus Authors
```

Comment:
I think it would be best to put this in the `pkg/value` as it can be reused in few different places and remove some duplicate code.
I think in the wal there was also a min/max functions which can be removed after this change.